src/plugin.ts

```diff
@@ -15,7 +15,7 @@ import * as promisify from 'promisify-node';
 
 interface PluginOptions {
   seleniumArgs?: string[];
-  seleniumJavaArgs?: string[];
+  javaArgs?: string[];
```

Comment:
I would tend to keep the previous name.

Reply:
But will it work? I see in selenium it's javaArgs and seleniumArgs. So maybe we should be consitent to make sure. Otherwise this won't work.